includes/class-wc-calypso-bridge-woocommerce-admin-navigation.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Modify the navigation to fit the bridge needs.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Calypso_Bridge_WooCommerce_Admin_Navigation
+ */
+class WC_Calypso_Bridge_WooCommerce_Admin_Navigation {
+
+	/**
+	 * Class Instance.
+	 *
+	 * @var WC_Calypso_Bridge_WooCommerce_Admin_Navigation instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! static::$instance ) {
+			static::$instance = new static();
+		}
+		return static::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_filter( 'woocommerce_navigation_root_back_url', array( $this, 'add_calypso_url' ) );
+	}
+
+	/**
+	 * Add calypso URL.
+	 */
+	public static function add_calypso_url() {
+		$strip_http = '/.*?:\/\//i';
+		$site_slug  = preg_replace( $strip_http, '', get_home_url() );
+		$site_slug  = str_replace( '/', '::', $site_slug );
```

Comment:
Is this line to catch the cases where WP might be installed in a subdirectory? Just looks vaguely familiar for calypso. If so, thinking we might not need that check since all wpcom sites have WP installed in the root. Also was wondering if there might be a way to get the site "slug" from an atomic option possibly.

Reply:
>Also was wondering if there might be a way to get the site "slug" from an atomic option possibly.

This would be great.  Does not look like there are any shorthand methods for grabbing it under https://github.com/Automattic/wpcomsh/blob/edc01bb249ebc0ca1aa6945319503997346ef764/class.atomic-plan-manager.php

Any ideas on how we might go about finding an option like this if it exists?

>Is this line to catch the cases where WP might be installed in a subdirectory?

I think it's to catch home URLs that may not be at the root and prevent an extra slash from being added to the URL.  E.g.,

`mysite.com/home` -> `mysite.com::home`